docs/Makefile

```diff
@@ -1,3 +1,10 @@
 clean:
 	rm -f src/sqlitegen/all.html src/sqlitegen/syntax_linkage.tcl
 
+
+spelings:
```

Comment:
It was deliberate - when, in 10 years' time, we run the AI spell checkers over all the comdb2 clones in the multiverse, there should only be one "orthographic issue " per repo....